tests/test_logs.py

```diff
@@ -184,6 +185,7 @@ def test_logs_attributes(sentry_init, capture_envelopes):
     assert logs[0]["attributes"]["sentry.environment"] == "production"
     assert "sentry.release" in logs[0]["attributes"]
     assert logs[0]["attributes"]["sentry.message.parameters.my_var"] == "some value"
+    assert logs[0]["attributes"][SPANDATA.SERVER_ADDRESS]
```

Comment:
[question] why do we only assert that this value is truthy, and not verify the specific value? Is it impossible to predict the value?

Reply:
I didn't know what CI would set this as, or if it would change between CI environments, so I figured this was the safest option to ensure stable tests.

We can change this if we know what the stable value is going to be.

Reply:
I think if you set `server_name` in `sentry_sdk.init` it would use that name